core/lib/spree/core/engine.rb

```diff
@@ -106,7 +107,10 @@ class Engine < ::Rails::Engine
         ]
 
         Rails.application.config.spree.adjusters = [
-          Spree::Adjustable::Adjuster::Promotion,
```

Comment:
@mad-eel on 2nd thought - let's not split this - we don't need a breaking change